app/_locales/en/messages.json

```
@@ -1594,6 +1594,9 @@
   "mainnet": {
     "message": "Ethereum Mainnet"
   },
+  "mainnetToken": {
+    "message": "This is a mainnet token address. We do not expect it to work on the current network..."
```

```suggestion
    "message": "This address matches a known mainnet token address. Recheck that you are on the right network and the contract address for the token you are trying to add..."
```
cc @rachelcope @danjm I think this copy is more clear. What do you think?

---

looks good to me